<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Calcolatore Limoncello</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="titolo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>5</y>
      <width>800</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <fontweight>Medium</fontweight>
     </font>
    </property>
    <property name="text">
     <string>Calcola le quantit√† giuste per il limoncello</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="sceltaCalcolo">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>70</y>
      <width>750</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Limoncello da un tentativo precedente</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Limoncello con un preciso obiettivo di gradazione alcolica e dolcezza</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Limoncello date le masse degli ingredienti</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="tentativoPrecedente" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <widget class="QComboBox" name="sceltaTentativo">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>0</y>
       <width>750</width>
       <height>30</height>
      </rect>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>Tentativo no. 1</string>
      </property>
     </item>
    </widget>
    <widget class="QDoubleSpinBox" name="massaScorze1">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>40</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="calcoloDaTentativo">
     <property name="geometry">
      <rect>
       <x>695</x>
       <y>40</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Calcola</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelScorze1">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>40</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Massa scorze</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tentativoObiettivo" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <widget class="QDoubleSpinBox" name="massaScorze">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>40</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="calcolaDaObiettivo">
     <property name="geometry">
      <rect>
       <x>695</x>
       <y>40</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Calcola</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelScorze2">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>40</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Massa scorze</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="gradazione">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>0</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="value">
      <double>30.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="labelGradazione">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>0</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Gradazione</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dolcezza">
     <property name="geometry">
      <rect>
       <x>695</x>
       <y>0</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.400000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="labelDolcezza">
     <property name="geometry">
      <rect>
       <x>615</x>
       <y>0</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Dolcezza</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tentativoDati" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <widget class="QPushButton" name="calcolaDaDati">
     <property name="geometry">
      <rect>
       <x>695</x>
       <y>40</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Calcola</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAcqua">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>40</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Acqua (g)</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAlcool">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>0</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Alcool (g)</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelZucchero">
     <property name="geometry">
      <rect>
       <x>615</x>
       <y>0</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Zucchero (g)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="massaZucchero">
     <property name="geometry">
      <rect>
       <x>695</x>
       <y>0</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="massaAlcool">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>0</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="massaAcqua">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>40</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="risultati" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>800</width>
      <height>100</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>741</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Il limoncello ottenuto con i dati che hai inserito ha le seguenti caratteristiche:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gradazione alcolica (%v/v):</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>190</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Dolcezza (g/mL):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelGradazioneAlcolica">
     <property name="geometry">
      <rect>
       <x>664</x>
       <y>40</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelDolcezza_2">
     <property name="geometry">
      <rect>
       <x>664</x>
       <y>70</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="risultati_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>800</width>
      <height>131</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>741</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Per ottenere questo limoncello avrai bisogno di:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Alcol (g):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
     </property>
    </widget>
    <widget class="QLabel" name="grammialcol">
     <property name="geometry">
      <rect>
       <x>664</x>
       <y>40</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="grammizucchero">
     <property name="geometry">
      <rect>
       <x>664</x>
       <y>70</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Zucchero (g):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Acqua (g):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
     </property>
    </widget>
    <widget class="QLabel" name="grammiacqua">
     <property name="geometry">
      <rect>
       <x>664</x>
       <y>100</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>sceltaTentativo</tabstop>
  <tabstop>massaScorze1</tabstop>
  <tabstop>sceltaCalcolo</tabstop>
  <tabstop>calcoloDaTentativo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
